### PR TITLE
UI Builder Initial Setup Improvements

### PR DESCRIPTION
**Pull request by wrpeck**
_Description of changes:_

Updating documentation on initial setup for using UI Builder components

### CODE REVIEW

File: src/pages/console/uibuilder/figmatocode.mdx [hunk 1]
```diff
@@ -30,25 +30,61 @@ After authenticating with Figma, you will see a list of components that you can
...
+
+In your project, run `amplify pull` to import React code generated from your Figma components. Then, import your components to use them in your application. Replace "ComponentA" and "ComponentB" with the names of your components:
+```jsx
 import { ComponentA, ComponentB }  from './ui-components';
 
 ```
 
+Amplify places all generated UI component code in the `src/ui-components` directory as JSX and TS files. All generated code is built with primitives from the Amplify UI library. 
+
+Although you can inspect the component code, any changes to component code will be overwritten on the next `amplify pull`. Instead, **[customize UI components in code](/console/uibuilder/override)**.
```
Comment (abdallahshaban557) [inline src/pages/console/uibuilder/figmatocode.mdx:84]: Maybe highlight a bit more what the link will let you change? It is a bit vague to provide a link before explaining that.

Comment (wrpeck) [reply src/pages/console/uibuilder/figmatocode.mdx]: Updated to clarify a bit more - thoughts on this?

**Review by abdallahshaban557 — DISMISSED** [review]
LGTM!

File: src/pages/console/uibuilder/figmatocode.mdx (hunk 1 above)
Comment (abdallahshaban557) [reply src/pages/console/uibuilder/figmatocode.mdx]: Better! approved it!

File: src/pages/console/uibuilder/figmatocode.mdx [hunk 2]
```diff
@@ -30,25 +30,61 @@ After authenticating with Figma, you will see a list of components that you can
...
+
+## Step 4: Import and render your components
+
+In your project, run `amplify pull` to import React code generated from your Figma components. Then, import your components to use them in your application. Replace "ComponentA" and "ComponentB" with the names of your components:
+```jsx
 import { ComponentA, ComponentB }  from './ui-components';
 
 ```
 
+Amplify places all generated UI component code in the `src/ui-components` directory as JSX and TS files. All generated code is built with primitives from the Amplify UI library.
```
Comment (rachelnabors) [inline src/pages/console/uibuilder/figmatocode.mdx:82]: Does it make sense here to say "Amplify CLI places..." to clarify which Amplify product is doing the thing?

File: src/pages/console/uibuilder/figmatocode.mdx [hunk 3]
```diff
@@ -30,25 +30,61 @@ After authenticating with Figma, you will see a list of components that you can
...
+
+![A component in edit mode with the Get component code button visible at the bottom](/images/studio/ui-setup1.png)
+
+Go to the **Initial project setup** tab and follow the setup instructions.
+
+![A pop up with the Initial Project Setup tab selected, showing code snippets needed to use Amplify Studio components](/images/studio/ui-setup2.png)
+
+## Step 4: Import and render your components
+
+In your project, run `amplify pull` to import React code generated from your Figma components. Then, import your components to use them in your application. Replace "ComponentA" and "ComponentB" with the names of your components:
```
Comment (rachelnabors) [inline src/pages/console/uibuilder/figmatocode.mdx:76]: This is confusing. You just said above to go to "Initial Project set up" tab. Then you say, "in your project, run..." People aren't in their project. Their in their IDE/terminal.

> From your project folder, run...

**Review by rachelnabors — CHANGES_REQUESTED** [review]

File: src/pages/console/uibuilder/figmatocode.mdx (hunk 3 above)
Comment (wrpeck) [reply src/pages/console/uibuilder/figmatocode.mdx]: TY, good catch. Fixed this.

File: src/pages/console/uibuilder/figmatocode.mdx (hunk 2 above)
Comment (wrpeck) [reply src/pages/console/uibuilder/figmatocode.mdx]: Agreed, added CLI

**Review by katiegoines — APPROVED** [review]
lgtm!

**Review by abdallahshaban557 — COMMENTED** [review]
Left a comment

File: src/pages/console/uibuilder/figmatocode.mdx [hunk 4]
```diff
@@ -30,25 +30,61 @@ After authenticating with Figma, you will see a list of components that you can
 
 ![figma](/images/console/ui-figma-sync.png)
 
-## Step 3: Pull UI components code in your project
+## Step 3: Configure your project
 
-In Studio, pick a component and choose **Configure** to get to the component editor screen. Choose **Get component code** at the bottom of the page.
+Before importing your components and theme into a project, the project will need an initial setup.
```
[the remark this answers is not in the document]
Comment (wrpeck) [reply src/pages/console/uibuilder/figmatocode.mdx]: Updated!

**Review by katiegoines — APPROVED** [review]

**Review by rachelnabors — APPROVED** [review]

**Pull request merged**